[Rewrite] Function Wikis - #1 (#201)
NilPointer-Software/bdfd-wiki: src/bdscript/userReacted.md

# $userReacted

> Checks if an user reacted to a message with the provided emoji. Returns "true", if the user did react else "false".

Reviewer: ```suggestion
Checks if a user reacted to a message with the provided emoji. Returns "true" if the user did react otherwise, "false".
```

## Syntax
```
$userReacted[Channel ID;Message ID;User ID;Emoji]
```

### Parameters
- `Channel ID` : The channel where the message is located.
- `Message ID` : The message to check the reactions for.

> - `User ID` : The user that should of reacted.
> - `Emoji` : The emoji that the user should of reacted with.

Reviewer: ```suggestion
- `User ID` : The user to check the reaction for.
- `Emoji` : The emoji of the reaction to check.
```
